### ISSUE TITLE
Coloring point clouds with file open

### ISSUE THREAD

**Issue by gitdFx** [CloudCompare/CloudCompare#2017]
Hi,
It would be great to be able to directly apply a coloring on point cloud when you open it.
I'm working on data filters, rendering the point cloud results in CC, with each filtering being a different point cloud.
So I would like to open every point cloud in CC and color every cloud with a different color, to easily visually compare the results of my filters.
Actually, I have to select each cloud and color it manually.

A command line arg would be great to do so.

**Comment by dgirardeau** [comment]
Yes, a command line argument to set the color could do the trick (but that would be a standard command, which means that you will have to load the cloud, set the color and then save it (as a BIN file or another format that supports colors).

Or maybe another way to do it is to use the Python plugin ([https://github.com/tmontaigu/CloudCompare-PythonRuntime](url)) and write a script that colorizes all the loaded entities in a more or less random way? I believe such a script would be easy to write).
